docs/sql/oracle/create-schema.sql

```diff
@@ -17,7 +17,7 @@
  */
 
 CREATE SEQUENCE "es_document_result_seq" MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 10 START WITH 1 CACHE 20;
-CREATE SEQUENCE "es_operation_template_seq" MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 10 START WITH 1 CACHE 20;
+CREATE SEQUENCE "es_operation_template_seq" MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20;
```

Comment:
Is this sequence new? Don't we need to update existing ones as well?

Reply:
I think this change requires a DB migration for both `enrollment-server` and `powerauth-cloud`.

Reply:
No, it does not. This is how it was until you changed it...

Reply:
I doubt it was me (I am just author of large merge commit), but I see, it's a mistake.